styles.css

```diff
@@ -44,11 +44,15 @@ label {
   height: 150px;
   margin: auto;
   display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
 #meditate-icon {
   width: 80px;
   margin: auto;
+  position: absolute;
```

Comment:
Some of the alignment issues you're having may be coming from having `position: absolute` set to some elements. Can you explain what you're hoping to achieve with this and we can see if there's a way to do it with flex so everything stays lined up?